.yarn/sdks/prettier/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "prettier",
-  "version": "2.8.1-sdk",
+  "version": "2.8.4-sdk",
```

Comment:
is this change needed?

Reply:
no... but also don't want to make an new PR for it!